source/process/mobile-release-process.md

```diff
@@ -0,0 +1,61 @@
+# Mobile App Release Process
+
+The Mattermost Mobile Apps team works on a monthly release process, with a new version submitted to the App Store and Google Play Store on the 5th of each month. Note: App Store approval may take a few days after the 5th. 
+
+## Release Timeline
+
+Notes:
+
+- All cut-off dates are based on 10am (San Francisco Time) on the day stated.
+- T-minus counts are measured in "working days" (weekdays other than major holidays in the US and Canada) prior to release day.
+
+### A. (T-minus 8 working days) Code Complete and Release Candidate Cut 
+
+1. PM:
+  - Submit pull request for changelog
+  - Check the minimum version number and submit pull request to update 
+2. Dev:
+  - Finish review of last pull requests for the current release and merge
+3. QA:
+  - Confirm all pull requests merged into the current release have been tested
```

Comment:
we should add here:
- create release branch for mattermost-mobile and mattermost-redux